templates/footer.php

```diff
@@ -0,0 +1,7 @@
+<footer class="amp-wp-footer" role="contentinfo" itemscope="itemscope" itemtype="http://schema.org/WPFooter">
+	<div>
+		<h2><?php echo esc_html( $this->get( 'blog_name' ) ); ?></h2>
+		<p><?php _e( 'Powered by WordPress.com', 'amp' ); ?></p>
```

Comment:
This should be "WordPress" but we should make it so WP.com can override as necessary.